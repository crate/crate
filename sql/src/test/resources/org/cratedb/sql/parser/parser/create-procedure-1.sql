CREATE PROCEDURE test.foo_bar(IN x INT, OUT y DOUBLE) LANGUAGE JAVA PARAMETER STYLE JAVA EXTERNAL NAME 'org.cratedb.ProcTest.fooBar'