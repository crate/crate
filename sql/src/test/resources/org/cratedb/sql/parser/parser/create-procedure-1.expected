org.cratedb.sql.parser.parser.CreateAliasNode@77925ae
aliasType: PROCEDURE
aliasInfo: (IN x INTEGER, OUT y DOUBLE) LANGUAGE java PARAMETER STYLE java
createOrReplace: false
javaClassName: org.cratedb.ProcTest
methodName: fooBar
name: test.foo_bar
statementType: CREATE PROCEDURE